src/masternode-payments.cpp

```diff
@@ -79,25 +79,25 @@ bool IsBlockValueValid(const CBlock& block, int nBlockHeight, CAmount blockRewar
 
     // we are synced, let's try to check as much data as we can
 
-    if(CSuperblockManager::IsSuperblockTriggered(nBlockHeight)) {
-        if(CSuperblockManager::IsValid(block.vtx[0], nBlockHeight, blockReward)) {
-            LogPrint("gobject", "IsBlockValueValid -- Valid superblock at height %d: %s", nBlockHeight, block.vtx[0].ToString());
-            // all checks are done in CSuperblock::IsValid, nothing to do here
-            return true;
-        }
+    if(sporkManager.IsSporkActive(SPORK_9_SUPERBLOCKS_ENABLED)) {
+        if(CSuperblockManager::IsSuperblockTriggered(nBlockHeight)) {
+            if(CSuperblockManager::IsValid(block.vtx[0], nBlockHeight, blockReward)) {
+                LogPrint("gobject", "IsBlockValueValid -- Valid superblock at height %d: %s", nBlockHeight, block.vtx[0].ToString());
+                // all checks are done in CSuperblock::IsValid, nothing to do here
+                return true;
+            }
 
-        // triggered but invalid? that's weird
-        if(sporkManager.IsSporkActive(SPORK_9_SUPERBLOCKS_ENABLED)) {
+            // triggered but invalid? that's weird
             LogPrintf("IsBlockValueValid -- ERROR: Invalid superblock detected at height %d: %s", nBlockHeight, block.vtx[0].ToString());
-            // should NOT allow invalid superblocks, when superblock enforcement is enabled
+            // should NOT allow invalid superblocks, when superblocks are enabled
             return false;
         }
-
-        // should NOT allow superblocks at all, when superblock enforcement is disabled
-        LogPrintf("IsBlockValueValid -- Superblock enforcement is disabled, no superblocks allowed\n");
+        LogPrint("gobject", "IsBlockValueValid -- No triggered superblock detected at height %d\n", nBlockHeight);
+    } else {
+        // should NOT allow superblocks at all, when superblocks are disabled
+        LogPrintf("IsBlockValueValid -- Superblocks are disabled, no superblocks allowed\n");
```

Comment:
I don't think this message makes sense here, since this will be hit for every block unless SPORK_9 is active.